index.html

```diff
@@ -0,0 +1,35 @@
+
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <title>Регистрация</title>
+</head>
+<body>
+    <h1>Регистрация</h1>
+    <form method="POST" action="/register">
+        {{ form.hidden_tag() }}
+        <p>{{ form.first_name.label }}<br>
+             {{ form.first_name(size=20) }}<br>
+            {% for error in form.first_name.errors %}{% endfor %}
+         </p>
+        <p>{{ form.last_name.label }}<br>
+             {{ form.last_name(size=20) }}<br>
+            {% for error in form.last_name.errors %}{% endfor %}
+        </p>
+
+        <p>{{ form.email.label }}<br>
+             {{ form.email(size=30) }}<br>
+             {% for error in form.email.errors %}{% endfor %}
+            </p>
+        <p>{{ form.password.label }}<br>
+             {{ form.password(size=30) }}<br>
+             {% for error in form.password.errors %}{% endfor %}
+            </p>
+        <p>{{ form.confirm_password.label }}<br>
+             {{ form.confirm_password(size=30) }}<br>
+             {% for error in form.confirm_password.errors %}{% endfor %}
+            </p>

```

Comment:
Можно заменить повторение кода. Попробуйте сделать так
```suggestion
        {% for field in form %}
            <p>
                {% if field.name not in ("submit", "csrf_token") %}
                {{ field.label }} <br>
                {{ field(size=20) }}
                {% for error in field.errors %}{% endfor %}
                {% endif %}
            </p>
        {% endfor %}
```